apps/explorer/app/contract/[id]/page.tsx

```diff
@@ -101,6 +111,7 @@ export default async function Page({ params }) {
   if (transactionError) throw transactionError
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +108]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Can parallelize this at the top with other requests.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/761?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->